src/git-repository.coffee

```diff
@@ -111,7 +116,14 @@ class GitRepository
       @repo.release()
       @repo = null
 
-    @subscriptions.dispose()
+    if @subscriptions?
```

Comment:
Are you sure on this one? That would make it inconsistent with the other things in `destroy()`.